pdns/recursordist/recursor_cache.cc

```diff
@@ -316,6 +316,20 @@ bool MemRecursorCache::entryMatches(MemRecursorCache::OrderedTagIterator_t& entr
   return match;
 }
 
+void MemRecursorCache::purge(const DNSName& qname, QType qt)
```

Comment:
This new method is not needed, see `MemRecursorCache::doWipeCache`
(it also neglects to wipe the ECS data)